pkg/hostagent/hostagent.go

```diff
@@ -568,6 +568,21 @@ const (
 	verbCancel  = "cancel"
 )
 
+func executeSSH(ctx context.Context, sshConfig *ssh.SSHConfig, port int, command ...string) error {
+	args := sshConfig.Args()
+	args = append(args,
+		"-p", strconv.Itoa(port),
+		"127.0.0.1",
+		"--",
+	)
+	args = append(args, command...)
+	cmd := exec.CommandContext(ctx, sshConfig.Binary(), args...)
+	if out, err := cmd.Output(); err != nil {
+		return fmt.Errorf("failed to run %v: %q: %w", cmd.Args, string(out), err)
+	}
+	return nil
+}
```

Comment:
Shouldn't we use `sshutil.SSHOpts` ?

Reply:
We are, it is getting passed in sshConfig as far as I know?

```go
        sshConfig := &ssh.SSHConfig{
                AdditionalArgs: sshutil.SSHArgsFromOpts(sshOpts),
        }
```